### PR TITLE
Reorganize arch

### PR DESCRIPTION
**Pull request by JEderonn**
This PR is to move architecture content to the PELUX Architecture repo, and only keep howto-sections in the Handbook.

### CODE REVIEW

**Comment by sashko** [comment]
You don't follow our commit message guidelines :/

**Comment by JEderonn** [comment]
> You don't follow our commit message guidelines :/

All commit messages have been updated. If you still have opinions, please be more specific.

**Review by sashko — COMMENTED** [review]
Software

**Comment by sashko** [comment]
> > You don't follow our commit message guidelines :/
> 
> All commit messages have been updated. If you still have opinions, please be more specific.

I don't wanna be too pedantic, but could you please at least use the present tense in titles?

**Comment by JEderonn** [comment]
> > > You don't follow our commit message guidelines :/
> > 
> > 
> > All commit messages have been updated. If you still have opinions, please be more specific.
> 
> I don't wanna be too pedantic, but could you please at least use the present tense in titles?

Got it, thanks.

**Comment by JEderonn** [comment]
> > Create top-level section 'Softare Update Management'
> 
> Software

Ah, good catch. Fixed

**Review by sashko — APPROVED** [review]

**Comment by TSAnsari** [comment]
Missing following part in commit messages,
Signed-off-by: <Name> <email>

Can be done by 
git commit -s --amend

**Comment by JEderonn** [comment]
> Missing following part in commit messages,
> Signed-off-by:
> 
> Can be done by
> git commit -s --amend

Will do tomorrow

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ A note on development images
 Both PELUX images are available as `-dev` variants, which include some extra
 development and debugging tools. For some platforms, those images are also
 available as `-update` to generate update artifacts for the :ref:`SOTA
-System<SOTA-system>`.
+System<getting-started-sota>`.
```
Comment (TSAnsari) [inline docs/chapters/baseplatform/building-PELUX-sources.rst:16]: The rst filename is getting-started-SOTA.rst
The reference has SOTA in small letters <getting-started-sota>. Does it work?

Comment (JEderonn) [reply docs/chapters/baseplatform/building-PELUX-sources.rst:16]: Yes, I don't think it refers to the filename but a label

Check https://pelux.io/jenkins/job/software-factory/view/change-requests/job/PR-174/lastSuccessfulBuild/artifact/PR-174/chapters/baseplatform/building-PELUX-sources.html

**Comment by JEderonn** [comment]
> > Missing following part in commit messages,
> > Signed-off-by:
> > Can be done by
> > git commit -s --amend
> 
> Will do tomorrow

Done

**Review by TSAnsari — APPROVED** [review]

**Review by sashko — APPROVED** [review]

**Pull request merged**